lib/pages/home.dart

```diff
@@ -1,170 +1,170 @@
+    final GitHubUser model;
+    const _InnerMyHomePage(this.model);
 
-  @override
-  Widget build(BuildContext context) {
-    return new Scaffold(
-        appBar: new AppBar(
-          title: new Text('GitHub Grass'),
-          actions: <Widget>[
-            new IconButton(
-              icon: new Icon(Icons.refresh),
-              onPressed: model.update,
-            )
-          ],
-        ),
-        body: new ListView.builder(
-          itemCount: monthlyContributions.length + 1,
-          itemBuilder: (BuildContext context, int index) {
-            if (index == 0) {
-              return _createIdentityWidget();
-            } else {
-              _ContributionTile monthlyContrib =
-                  monthlyContributions[index - 1];
-              return _createMonthlyContributionMap(monthlyContrib);
-            }
-          },
-        ));
+    @override
+    _MyHomePageState createState() => new _MyHomePageState(model);
   }
 
-  Widget _createIdentityWidget() {
-    DateTime today = new DateTime.now();
-
-    Contribution c = model.contributions.firstWhere((c) =>
-        c.year == today.year && c.month == today.month && c.day == today.day);
-    int value = c.value ?? 0;
-
-    String todayLabel =
-        '${today.year.toString()}-${today.month.toString().padLeft(2,'0')}-${today.day.toString().padLeft(2,'0')}';
-
-    return new Padding(
-      padding: const EdgeInsets.symmetric(vertical: 10.0),
-      child: new Center(
-        child: new Column(
-          children: <Widget>[
-            new CircleAvatar(
-              backgroundImage: new NetworkImage(
-                  'https://avatars.githubusercontent.com/${model.username}'),
-            ),
-            new Text(
-              model.username,
-              style: new TextStyle(
-                fontSize: 20.0,
-              ),
-            ),
-            value > 0
-                ? new Text(
-                    'Today\'s contribution is ${value}! 👍',
-                    style: new TextStyle(
-                      color: Colors.green,
-                      fontWeight: FontWeight.bold,
-                    ),
-                  )
-                : new Text(
-                    '${todayLabel} is not contributed yet.❌',
-                    style: new TextStyle(
-                      color: Colors.red,
-                    ),
-                  ),
-          ],
-        ),
-      ),
-    );
+  class _ContributionTile {
+    final int year;
+    final int month;
+    final String yearMonth;
+    final List<int> contributions;
+
+    _ContributionTile({this.year, this.month})
+        : contributions = [],
+          yearMonth = '${year.toString()}-${month.toString().padLeft(2,'0')}';
+
+    bool isPresentMonth() {
+      DateTime today = new DateTime.now();
+      return (year == today.year && month == today.month);
+    }
+
+    @override
+    String toString() => '${yearMonth}:' + contributions.length.toString();
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Avoid using braces in interpolation when not needed.